doc/plugins/unittest.rst

```diff
@@ -0,0 +1,69 @@
+###############
+Spyder Unittest
+###############
+
+**Spyder-unittest** is a plugin that integrates popular unit test frameworks with Spyder, allowing you to run test suites and view the results in the IDE.
+
+.. image:: /images/unittest/unittest-run.png
+   :alt: Spyder Unittest in Spyder
+
+Using unittest inside Spyder allows you to take advantage of their web interface alongside Spyder’s powerful features such as the Variable explorer, console and debugger.
```

Comment:
```suggestion
Using unittest inside Spyder allows you to take advantage of their web interface alongside Spyder’s powerful features such as the Variable explorer, console and debugger.
```
This is left from the notebook one and I'm not sure it applies to this one but @ccordoba12 can confirm.

Reply:
Yeah, it doesn't apply here.

Reply:
this was removed